services/opm/opm.service.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+const { addv } = require('../text-formatters')
+const { BaseService, NotFound, InvalidResponse } = require('..')
+
+module.exports = class Luarocks extends BaseService {
```

Comment:
Will need to rename this class as we've already got an existing `Luarocks` service. Maybe `OpmVersion`? It'd probably be best to rename the file to something like `opm-version.service.js` too
https://github.com/badges/shields/blob/d5af9f884b6db2f4bba9c2582123d08a2c97486a/services/luarocks/luarocks.service.js#L17

Reply:
Ah yes that was a typo... I've renamed the file and class name.